ci/functional/test_main.sh

```diff
@@ -47,7 +47,8 @@ test_cluster() {
         NODELIST=${tnodes}                              \
         BUILD_URL=\"${BUILD_URL:-Unknown in GHA}\"      \
         STAGE_NAME=\"$STAGE_NAME\"                      \
-        $(cat ci/functional/test_main_prep_node.sh)"
+        COVFN_DISABLED=\"$COVFN_DISABLED\"              \
+	$(cat ci/functional/test_main_prep_node.sh)"
```

Comment:
This is on the wrong line.  The export should be added before calling `$(cat ci/functional/test_main_node.sh)"`.  Also be careful with tabs and spaces.